catalog/dags/maintenance/pr_review_reminders/pr_review_reminders.py

```diff
@@ -57,6 +66,16 @@ class ReviewDelta:
     days: int
 
 
+@task
+def get_maintainers(github_pat: str) -> set[str]:
+    gh = GitHubAPI(github_pat)
+    maintainer_info = gh.get_team_members(MAINTAINER_TEAM)
+    maintainers = {
+        member["login"] for member in maintainer_info
+    } ^ NON_MAINTAINER_IGNORATIONS
```

Comment:
If the set of maintainer logins cannot overlap with `NON_MAINTAINER_IGNORATIONS`, can we use `|` instead?

```suggestion
    } | NON_MAINTAINER_IGNORATIONS
```

Reply:
Ugh, thanks for correcting this 😅 I knew there was an operator I wanted and started off with `&` (much to my incorrect dismay). This is the right one for sure!